react_main/build/asset-manifest.json

```diff
@@ -1,26 +1,26 @@
 {
   "files": {
     "main.css": "/static/css/main.a2ca88fb.chunk.css",
```

Comment:
always react build in a separate commit, you can disable squash merging if you don't want to make another PR

Reply:
I see, it would probably best for a "checkpoint" in commit list for what front end changes have been built

I started combining it with new PRs because having a separate PR to be approved for a react build after front-end changes seemed a bit tedious.

I think I'll probably create a PR for react builds if it follows multiple non-urgent front-end changes, otherwise an unsquashed PR with build for urgent hotfixes and/or large front-end changes from here on.

Thank you for the suggestion!